lib/Property.php

```diff
@@ -37,7 +37,7 @@ abstract class Property extends Node
     /**
      * List of parameters.
      *
-     * @var array
+     * @var Parameter[]
```

Comment:
I think we should go one step further:
```suggestion
     * @var array<string, Parameter>
```

Reply:
I like that as well, yet I'm not sure how "official" that phpdoc syntax is.

Reply:
its kind of proprietary syntax which is used in psalm, phpstan and most static analysis tooling.

we could also use a more tool targetting syntax, so we don't kill the "offical form":

```suggestion
     * @phpstan-var array<string, Parameter>
     * @var Parameter[]
```

Reply:
If you prefer this then I can add it